types/jasmine/ts3.1/index.d.ts

```diff
@@ -13,6 +13,7 @@
 //                 Stephen Farrar <https://github.com/stephenfarrar>
 //                 Mochamad Arfin <https://github.com/ndunks>
 //                 Alex Povar <https://github.com/zvirja>
+//                 Dominik Ehrenberg <https://github.com/djungowski>
```

Comment:
Notice, you should put your name here only if you would like to become a maintainer of this project and stay here for a long period of time. You will be notified each time there is a change related to this project.

So consider yourself whether that's exactly what you would like to sign in :wink:

Reply:
Since I use jasmine a lot, that would be fine for me - I guess 😅